.github/workflows/ci.yml

```diff
@@ -0,0 +1,73 @@
+name: C++ CI Workflow
+
+on: [push]
+
+jobs:
+  build:
+    name: '[${{ matrix.os }}@${{ matrix.build_type }}]'
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        build_type: [Debug, Release]
+        os: [ubuntu-latest, windows-latest, macOS-latest]
+
+    steps:
+    - uses: actions/checkout@master
+        
+    # Print environment variables to simplify development and debugging
+    - name: Environment Variables
+      shell: bash
+      run: env
+        
+    # ============
+    # DEPENDENCIES
+    # ============
+    
+    # Waiting a github action that installs dependencies in a
+    # multiplatform setup (likely using TS or JS), we install
```

Comment:
What do `TS` and `JS` stand for?

Reply:
Typescript and Javascript. There are two types of Actions: docker-based and js-based. The former run isolated and can only operate on the `${GITHUB_WORKSPACE}` directory, the latter instead run in the host VM.

For instance, I tried to develop an Action to install dependencies in a multi-os setting (similar to the Travis addons). The only type to install deps that are persistent among steps is js, something I'm not really familiar with :/